go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/pingcap/tipb
+
+go 1.15
+
+require (
```

Comment:
I think we'd better stick to the same version requirement as https://github.com/pingcap/kvproto/blob/master/go.mod

Reply:
tipb has it's own requirements. go modules always pick the latest same major version while build, so we only need to take care don't intreduce higher versions.

Reply:
Actually I'm thinking whether it is possible to make kvproto & tipb identical. I think they are the same and currently I cannot think out reasons to maintain two different approaches.